src/frameworks/reactively.ts

```diff
@@ -16,7 +16,10 @@ export const reactivelyFramework: ReactiveFramework = {
       read: () => r.get(),
     };
   },
-  effect: (fn) => new Reactive(fn, true),
+  effect: (fn) => {
+    fn();
```

Comment:
Why is this needed? Eagerly evaluating effects should probably be done by calling the required function in the framework (stabilize) instead of calling the effect function twice (which gives reactively and oby a disadvantage if the effect is slow because they can't memoize it)

Reply:
the new unit tests that I added to ensure we're comparing apples to apples across the different frameworks only work with this added.

happy to change it, but I feel like it's important for the unit tests across all frameworks to work the same way.

Reply:
I agree with having tests to ensure that the benchmark is fair, although I think the unit tests may be more strict than we expect. Afaik we don't have any tests that read or write during the build stage (they always return functions that are run outside of the build stage).

Then, making withBuild run a stabilize tick after construction should get the effects to run as intended

Reply:
This seems to pass tests for me

```ts
export const reactivelyFramework: ReactiveFramework = {
  name: "@reactively",
  signal: (initialValue) => {
    const r = new Reactive(initialValue);
    return {
      write: (v) => r.set(v),
      read: () => r.get(),
    };
  },
  computed: (fn) => {
    const r = new Reactive(fn);
    return {
      read: () => r.get(),
    };
  },
  effect: (fn) => {
    return new Reactive(fn, true);
  },
  withBatch: (fn) => {
    fn();
    stabilize();
  },
  withBuild: (fn) => {
    const out = fn();
    stabilize();
    return out;
  },
};
```